packages/block-editor/src/hooks/duotone.js

```diff
@@ -302,12 +302,12 @@ function useDuotoneStyles( {
 			// `inline-block` is used instead of `none` to ensure that scroll
 			// position is not affected, as `none` results in the editor
 			// scrolling to the top of the block.
-			blockElement.style.display = 'inline-block';
+			blockElement.style.setProperty( 'display', 'inline-block' );
```

Comment:
Glad `CSSStyleDeclaration` has a property set method 👍 I wish it was the same for all those other `element` properties.